src/components/aboutModal/aboutModal.tsx

```diff
@@ -93,6 +93,16 @@ const AboutModal: React.FC<AboutModalProps> = ({
               </TextListItem>
             </React.Fragment>
           )}
+          {stats?.platform.machine && (
```

Comment:
Technically, this may be an empty string. Do I need to add another condition to prevent it from showing, or does TypeScript automatically do the right thing here?

CC @nicolearagao

Reply:
Empty string is considered falsy, the expression will short-circuit and nothing will be rendered. U are good!